Testing/functional/e2e/cypress/integration/timeline/errors.js

```diff
@@ -0,0 +1,31 @@
+const { AuthMethod } = require("../../support/constants")
+
+describe('Banner Error', () => {
+    before(() => {
+        cy.intercept("GET", "/v1/api/Note/*", (req) => {
+            req.reply((res) => {
+                res.body.resultStatus = 0;
+                res.body['resultError'] = {
```

Comment:
lets use a fixture to return mock data

Reply:
as we dont have a lot of data to mock here, I don't think fixture is better. Personally, I prefer to see my test data included here as it is clearer, unless we have some good arguments for fixture vs. this?
Any thoughts @sslaws @FuriousLlama

Reply:
I like the idea that the mock data is stored in a file vs in code. +1 on using fixtures.

Reply:
mock data has been moved